calico/getting-started/kubernetes/helm.md

```diff
@@ -55,14 +55,22 @@ echo '{ installation: {kubernetesProvider: EKS }}' > values.yaml
 
 #### Install {{site.prodname}}
 
+1. Create the `tigera-operator` namespace.
+
+   ```
+   kubectl create namespace tigera-operator
```

Comment:
I thought with the `--namespace tigera-operator` I thought helm would create that if needed. But I'm just going off what I remember understanding so there is big room for me being wrong.

Reply:
I believe you can also pass `--create-namespace` to helm to allow it to create the namespace for you, but I like this because it's explicit and gives users a chance to pre-provision anything they need / configure the namespace before installing the chart.